ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -51,37 +51,37 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseEventID": "placementGatekeeping",
-    "UserRole": "[LASOLICITOR]",
-    "CRUD": "CRU"
+    "CaseEventID": "placement",
+    "UserRole": "caseworker-publiclaw-solicitor",
+    "CRUD": "R"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseEventID": "placement-PREPARE_FOR_HEARING",
+    "CaseEventID": "placementGatekeeping",
     "UserRole": "[LASOLICITOR]",
     "CRUD": "CRU"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseEventID": "placement",
-    "UserRole": "caseworker-publiclaw-solicitor",
+    "CaseEventID": "placement-PREPARE_FOR_HEARING",
+    "UserRole": "[LASOLICITOR]",
     "CRUD": "CRU"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "placementGatekeeping",
     "UserRole": "caseworker-publiclaw-solicitor",
-    "CRUD": "CRU"
+    "CRUD": "R"
```

Comment:
I am suprised to see changes to this file. Nothing to do with la barrister 🤔

Reply:
Yes i changed this file because caseworker-publiclaw-solicitor is a general idam role given to all solicitors and if we put things like placementGatekeeping CRU permissions on the general idam role it means that [LASOLICITOR] and [LABARRISTER] can both see these events which shouldn't be the case - hence why i've changed it to remove [LABARRISTER] from seeing this event